### PR TITLE
Fix more unused variable, method warnings reported by eclipse (#113)

### PR DESCRIPTION
**Pull request by amcp**

### CODE REVIEW

File: janusgraph-core/pom.xml [hunk 1]
```diff
@@ -66,12 +66,10 @@
         <dependency>
             <groupId>commons-collections</groupId>
             <artifactId>commons-collections</artifactId>
-            <version>${commons.collections.version}</version>
         </dependency>
         <dependency>
             <groupId>commons-configuration</groupId>
             <artifactId>commons-configuration</artifactId>
-            <version>1.10</version>
```
Comment (jerryjch) [inline janusgraph-core/pom.xml]: Please verify all the removed lines here are defined in the top level with consistent versions.

Comment (amcp) [inline janusgraph-core/pom.xml]: line 546 in master pom, same version

File: janusgraph-core/pom.xml [hunk 2]
```diff
@@ -80,7 +78,6 @@
         <dependency>
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
-            <version>1.7</version>
```
Comment (amcp) [inline janusgraph-core/pom.xml]: line 820 in master pom, same version

File: janusgraph-core/pom.xml [hunk 3]
```diff
@@ -93,7 +90,6 @@
         <dependency>
             <groupId>com.github.stephenc.high-scale-lib</groupId>
             <artifactId>high-scale-lib</artifactId>
-            <version>1.1.4</version>
```
Comment (amcp) [inline janusgraph-core/pom.xml]: in master pom line 938 - older version 1.1.2 so upgraded master pom to 1.1.4

File: janusgraph-core/pom.xml [hunk 4]
```diff
@@ -136,7 +132,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
-                <version>2.5</version>
```
Comment (amcp) [inline janusgraph-core/pom.xml]: in master pom at line 329 but was 2.4 so upgraded to 2.5

**Pull request merged**